### ISSUE TITLE
Custom element doesn't exist: vertical-stack-in-card

### ISSUE THREAD

**Issue by damiano75** [ofekashery/vertical-stack-in-card#91]
Home Assistant 0.110.3
HassOS 3.13
Supervisor 224

After upgrade to the last version of vertical-stack-in-card error "Custom element doesn't exist: vertical-stack-in-card" in every card and using browser Chrome, Edge and Firefox and iOS app on iPhone

![image](https://user-images.githubusercontent.com/47896789/82915684-5b4b3e00-9f71-11ea-880c-d9bd2d264ff5.png)

I reinstalled 0.3.2 version and the problem is gone

**Comment by gadgetchnnel** [comment]
I had the same issue but, when I checked, I found that vertical-stack-in-card had disappeared from my Lovelace resources.
I don't know if it is just a coincidence but I also found that stack-in-card (which I had been testing out) was appearing twice in my Lovelace resources. I deleted the second stack-in-card, and uninstalled and reinstalled vertical-stack-in-card version 0.3.3 and it started working again.
